conversions/src/Data/Conversions.hs

```diff
@@ -225,12 +225,12 @@ convertErrorFromNatural value =
   maybe (throwError $ UserBoundError value minBound maxBound) pure
     $ checkedFromIntegral value
 
-convertErrorFromIntegral
+convertErrorFromInt
```

Comment:
Comment: I am now thinking `converErrorFromSizedInt` may be a better pick.

Reply:
I do not agree the current name is an improvement, as `Int` implies `Data.Int(Int)` which has nothing to do with this specific instance. What about `convertErrorBounded`?

Reply:
@mbj you are right. However, we also have `convertBoundedFromIntegral` which is quite similar to `convertErrorBounded` implementation-wise with minor changes. I see no obvious course of action for now, something to think about.